### PR TITLE
Community - Match width of editor preview and actual post

### PR DESCRIPTION
**Pull request by roadscape**
From @quochuy #3063:
> Changing the width of editor preview to 50rem like PostFull

### CODE REVIEW

**Comment by roadscape** [comment]
Had to change max-width to `42rem` because PostFull limits to `40rem` and the preview adds `1rem` on each side.

**Comment by quochuy** [comment]
@roadscape I noticed the width has been changed from 50rem to 42rem which is very small as opposed to the 50rem of the PostFull width.

File: src/app/components/elements/ReplyEditor.scss [hunk 1]
```diff
@@ -87,7 +87,7 @@
 }
 
 .ReplyEditor .Preview {
-  max-width: 50rem;
+  max-width: 42rem;
```
Comment (quochuy) [inline src/app/components/elements/ReplyEditor.scss]: Why this drastic reduction of the max-width value? PostFull was at 50rem.

Comment (quochuy) [reply src/app/components/elements/ReplyEditor.scss]: Unless you wanted to make it 52rem and remove my margins and replace them with a padding of 1em on either side?

Comment (quochuy) [reply src/app/components/elements/ReplyEditor.scss]: OK, gotcha.
42.125rem is actually better as it would render to exactly 640px

Comment (quochuy) [reply src/app/components/elements/ReplyEditor.scss]: Screen Shot 2019-05-03 at 7 34 27 pm

Comment (quochuy) [reply src/app/components/elements/ReplyEditor.scss]: I opened a new PR

**Pull request merged**